### ISSUE TITLE
[RotaryCraft v33a] Wooden gearbox doesn't get damaged

### ISSUE THREAD

**Issue by Tensor6** [ReikaKalseki/Reika_Mods_Issues#3280]
The grinder managed to fill up the reservoir and the gearbox is still not damaged.

https://github.com/ReikaKalseki/Reika_Mods_Issues/assets/47905087/a788ed60-a7b6-4891-a82b-aeb086ac5c7d

**Comment by tprussak** [comment]
Wooden gearboxes now only get damaged if they overheat, which typically happens if they are transferring too much power.

**Comment by Tensor6** [comment]
Yes I know it'll get damaged once it reaches 100C but the gearbox is transferring 16 kW and it's not heating up at all. This issue started happening when I updated to v32 then persisted in v33a.
